Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Collections/GridObjectCollection.cs

```diff
@@ -49,7 +49,35 @@ public OrientationType OrientType
         public LayoutOrder Layout
         {
             get { return layout; }
-            set { layout = value; }
+            set 
+            { 
+                layout = value; 
+                UpdateCollection();
+            }
+        }
+
+        public enum AnchorType
+        {
+            UpperLeft,
+            UpperCenter,
+            UpperRight,
+            MiddleLeft,
+            MiddleCenter,
+            MiddleRight,
+            BottomLeft,
+            BottomCenter,
+            BottomRight
+        };
+
+        [SerializeField, Tooltip("Where the grid is anchored relative to local origin")]
+        private AnchorType anchor = AnchorType.MiddleCenter;
+        public AnchorType Anchor {
+            get { return anchor;}
+            set
+            {
+                anchor = value;
+                UpdateCollection();
```

Comment:
nit: only UpdateCollection on value change

Reply:
I think the same comment applies to the LayoutOrder one too!

Reply:
Thought about why the other fields don't call UpdateCollection() after and realized this is because you may want to change multiple properties in a frame, and only need to recompute once. Decided to follow that existing pattern to save on layout compute.